README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
@raeeschachar Yes. I meant that 'paver e2e_test' should exclude the whitelabel tests without using the '--exclude' flag. I am thinking the case where you want to run both normal and whitelable tests in one command is very rare, but I may be wrong. Thoughts?